src/Compilers/CSharp/Portable/Binder/PatternSwitchBinder2.cs

```diff
@@ -231,4 +239,57 @@ internal override void BindPatternSwitchLabelForInference(CasePatternSwitchLabel
             }
         }
     }
+
+    partial class BoundPatternSwitchStatement2
+    {
+        public HashSet<LabelSymbol> ReachableLabels => this.DecisionDag.ReachableLabels;
+    }
+
+    partial class BoundDecisionDag
+    {
+        HashSet<LabelSymbol> _reachableLabels;
```

Comment:
>HashSet<LabelSymbol> _reachableLabels; [](start = 8, length = 38)

Please add explicit accessibility modifier

Reply:
Addressed in https://github.com/dotnet/roslyn/pull/24059/commits/114a56e9077ed03f08443889b69cae6972a4acc6 of https://github.com/dotnet/roslyn/pull/24059